src/Square.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './style.css';
+
+function Squares(props) {
```

Comment:
One small thing - you can use object destructuring here if you like to avoid needing to call `props.` multiple times. 

```js
function Square({click, value})
```